README.md

```diff
@@ -5,18 +5,12 @@
 [![npm](https://img.shields.io/npm/dt/react-image.svg?style=flat-square)](https://www.npmjs.com/package/react-image)
 [![npm](https://img.shields.io/npm/dm/react-image.svg?style=flat-square)](https://www.npmjs.com/package/react-image)
 [![Known Vulnerabilities](https://snyk.io/test/github/mbrevda/react-image/badge.svg)](https://snyk.io/test/github/mbrevda/react-image)
-[![wercker status](https://app.wercker.com/status/51bfd9b8aa6e52acf77310e17f00aff4/s/master 'wercker status')](https://app.wercker.com/project/byKey/51bfd9b8aa6e52acf77310e17f00aff4)
-[![codecov](https://codecov.io/gh/mbrevda/react-image/branch/master/graph/badge.svg)](https://codecov.io/gh/mbrevda/react-image)
 
-**React Image** is an `<img>` tag replacement for [React.js](https://facebook.github.io/react/), featuring preloader and multiple image fallback support.
+**React Image** is an `<img>` tag replacement and hook for [React.js](https://facebook.github.io/react/), supporting fallback to alternate sources when loading an image fails.
 
-With **React Image** you can specify multiple images to be used as fallbacks in the event that the browser couldn't load the previous image. Additionally, you can specify any React element to be used before an image is loaded (i.e. a spinner) and in the event than the specified image(s) could not be loaded.
+**React Image** allows one or more images to be used as fallback imagess in the event that the browser couldn't load the previous image. When using the componenet, you can specify any React element to be used before an image is loaded (i.e. a spinner) and in the event than the specified image(s) could not be loaded. When using the hook this can be achives by wrapping the compomenet with [`<Suspense>`](https://reactjs.org/docs/react-api.html#reactsuspense) and specifying the `fallback` prop.
```

Comment:
```suggestion
**React Image** allows one or more images to be used as fallback images in the event that the browser couldn't load the previous image. When using the component, you can specify any React element to be used before an image is loaded (i.e. a spinner) and in the event that the specified image(s) could not be loaded. When using the hook this can be achieved by wrapping the component with [`<Suspense>`](https://reactjs.org/docs/react-api.html#reactsuspense) and specifying the `fallback` prop.
```